### ISSUE TITLE
[Accessibility][Object Information - Create from Template]: Screen reader announce incorrect information as 'invalid' when focus land on first template.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2020]
**User Experience:**
Keyboard dependent users would face difficulty if focus does not move to the dialog box that opens up after selecting any control.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. Navigate on homepage with tab key up to 'New' button and press enter on it.
3. 'Create from scratch' pop up will open.
4. Select 'Create from Template' radio button and then press tab key to land on its first template i.e. Echo bot.
5. Observe the issue with screen reader.

**Pre-Requisite:**
First template i.e. Echo bot should not be selected.

**Actual Result:**
Screen reader announce incorrect information as 'invalid' when focus land on first template. Screen reader read as 'Echo bot radio button non selected required invalid'.

**Expected Result:**
Screen reader should not announce incorrect information as 'invalid' when focus land on first template.

**Note:**
This issue is repro only once when first template is not selected.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/CELA/Official_Microsoft_Accessibility_Standards/MAS%204.2.1%20%E2%80%93%20Object%20Information.docx?d=weee97bbd999048a08b036ea775581aa8&csf=1&e=b4HOKE

**Attachment:**
[MAS4.2.1_Choose from Template.zip](https://github.com/microsoft/BotFramework-Composer/files/4224762/MAS4.2.1_Choose.from.Template.zip)

**Comment by liweitian** [comment]
@corinagum This should be solved due to UI changed

**Comment by beyackle** [comment]
Double-checked this, and I can't repro it. I think it's been solved already.

**Comment by ashish315** [comment]
As UI is updated and bug is resolved. So, HCL closing this issue.
